feature-group-definitions/scheduler.yml

```diff
@@ -1,3 +1,4 @@
+name: "Scheduler API"
```

Comment:
API because scheduling is a browser-internal thing as well. Similar argument to Navigation API.

Reply:
> browser-internal thing

Do we have any evidence that web developers know or care about this fact?

Reply:
No, I don't think most web developers think about how work is scheduled unless they specialize in performance. But it's like "paint", "layout" or "fetch". Browsers do these things, but there are also APIs for them.